generate-certificates.sh

```diff
@@ -38,6 +38,13 @@ if [[ "${database,,}" != "cassandra" && "${database,,}" != "elastic" && "${datab
    exit 2
 fi
 
+# Setting the PATH variable for OpenSearch
+if [[ "${database,,}" == "opensearch" ]]; then
+   PATH+=:/usr/share/opensearch/jdk/bin/
+   echo "When choosing OpenSearch, PATH is modified to be able to sign certificates."
```

Comment:
I would change this to "When choosing OpenSearch, PATH is temporarly modified to include java keytool"